main.py

```diff
@@ -12,6 +12,138 @@
+
+    # to reduce the complexity of computations in neural network
+    x_train = x_train / 255
+    x_test = x_test / 255
+
+    return (x_train, y_train), (x_test, y_test)
+
+def seq_model():
+    """
+    Returns convolutional neural network with a sequential model
+
+    Parameters
+    ----------
+    None
+
+    Returns
+    -------
+    model: keras.Sequential
+        Convolutional neural network with a sequential model
+    """
+    model = keras.Sequential()
+
+    # Instantiate 32 x 32 images as a Keras tensor
+    model.add(keras.Input(shape=(32, 32, 3)))
+
+    # Add convolutional layers, 32 -> 64 filters each, with relu activation for a simple linear NN
+    # Block 1 of Layers
+    model.add(layers.Conv2D(32, kernel_size=(3,3), strides=(2, 2), activation='relu'))
+    model.add(layers.MaxPooling2D(2))
+
+    # Block 2 of Layers
+    model.add(layers.Conv2D(64, kernel_size=(3,3), activation='relu'))
+    model.add(layers.MaxPooling2D(2))
+    model.add(layers.GlobalAveragePooling2D())
+    # Process data using Dense operator
+    model.add(layers.Dense(128, activation='relu'))
+    model.add(layers.Dense(10, activation='softmax'))
+
+    # Flatten tensor shape to one-dimension with average pooling to avoid overfitting
+    #model.add(layers.GlobalAveragePooling2D())
+    
+    # Compile model with adam algorithm due to large dataset, and a cross-entropy loss function
+    model.compile(loss='sparse_categorical_crossentropy', optimizer='adam', metrics=['accuracy'])
+    
+    return model
+
+def plot_model(history):
+    """
+    Plots validation, accuracy, and loss data of the CNN over epochs
+
+    Parameters
+    ----------
+
+    History object returned by the fit method applied to the CNN
+
+    Returns
+    -------
+    plt: Pyplot by Matplotlib
+        Graphs containing loss, validation, and accuracy functions
+    """
+    
+    # Summarize history for accuracy
+    plt.plot(history.history['accuracy'])
+    plt.plot(history.history['val_accuracy'])
+    plt.title('Model Accuracy')
+    plt.ylabel('accuracy')
+    plt.xlabel('epoch')
+    plt.legend(['train', 'test'], loc='upper left')
+    plt.show()
+    
+    # Summarize history for loss
+    plt.plot(history.history['loss'])
+    plt.plot(history.history['val_loss'])
+    plt.title('Model Loss')
+    plt.ylabel('loss')
+    plt.xlabel('epoch')
+    plt.legend(['train', 'test'], loc='upper left')
+    plt.show()
+
+def cnn_main():
+    """
+    Calls data loading, sequential model, and plotting functions
+
+    Parameters
+    ----------
+    None
+
+    Returns
+    -------
+    Pyplot by Matplotlib
+    """
+    # Call data loading function
+    (x_train, y_train), (x_test, y_test) = cifar_load()
+
+    # Call sequential model function to retrieve model
+    model = seq_model()
+    
+    # Fit the model to create history object
+    history = model.fit(x_train, y_train, validation_data=(x_test, y_test), validation_split=0.33, epochs=50, batch_size=10, verbose=1)
```

Comment:
just wondering why you chose validation split as 0.33?!

Reply:
batch size in the future should also be multiples of 2^n (e.g 128,256, 512, 1024...)